packages/create-svelte/templates/default/src/routes/sverdle/+page.svelte

```diff
@@ -0,0 +1,262 @@
+<script lang="ts">
+	import Keyboard from './Keyboard.svelte';
+	import { confetti } from '@neoconfetti/svelte';
+	import { enhance } from '$app/forms';
+	import { browser } from '$app/environment';
+	import type { PageData, ActionData } from './$types';
+
+	/** @type {import('./$types').PageData} */
+	export let data: PageData;
+
+	/** @type {import('./$types').ActionData} */
+	export let form: ActionData;
+
+	$: i = data.answers.length;
+	$: won = data.answers.at(-1) === 'xxxxx';
+
+	/** @type {Record<string, 'exact' | 'close' | 'missing'>}*/
+	let keys: Record<string, 'exact' | 'close' | 'missing'> = {};
```

Comment:
It feels a little inconcistent that the states are written out for the keys but the guesses use the shortform.

Reply:
It's basically because `exact` and `close` and `missing` are used as classnames (and only as classnames). I've moved this logic into `Keyboard.svelte` to (hopefully) make that clearer